src/dropdown/list/dropdown-list.component.ts

```diff
@@ -67,7 +67,7 @@ import { ScrollCustomEvent } from "./scroll-custom-event.interface";
 				[attr.aria-selected]="item.selected"
 				[id]="getItemId(i)"
 				[attr.title]=" showTitles ? item.content : null"
-				[attr.disabled]="item.disabled"
+				[attr.disabled]="item.disabled ? '' : undefined"
```

Comment:
```suggestion
				[attr.disabled]="item.disabled ? true : undefined"
```

This will make it easier to read. Other than that, looks good to me!

Reply:
Ok. I see other examples in the code base using `[attr.disabled]="item.disabled ? true : null"`, so I changed it to that.

Reply:
That's even better! 😄